### ISSUE TITLE
QoRTs localisation problem

### ISSUE THREAD

**Issue by eanoz** [ewels/MultiQC#1153]
**Description of bug:**
I was trying to make a whole report of RNAseq data including QoRTs reports.

**MultiQC Error log:**
Module qorts raised an exception:
```python
Traceback (most recent call last):
  File "/Users/Ernesto/opt/miniconda3/lib/python3.7/site-packages/multiqc/multiqc.py", line 546, in run
    output = mod()
  File "/Users/Ernesto/opt/miniconda3/lib/python3.7/site-packages/multiqc/modules/qorts/qorts.py", line 29, in __init__
    self.parse_qorts(f)
  File "/Users/Ernesto/opt/miniconda3/lib/python3.7/site-packages/multiqc/modules/qorts/qorts.py", line 67, in parse_qorts
    self.qorts_data[s_name][s[0]] = float(s[i+1])
ValueError: could not convert string to float: '1,22'
```

**File that triggers the error:**
[QC.summary.txt](https://github.com/ewels/MultiQC/files/4433284/QC.summary.txt)

**MultiQC run details (please complete the following):**
 - Command used to run MultiQC: `~/opt/miniconda3/bin/multiqc . --dirs --interactive -o QC`
 - MultiQC Version: last version
 - Operating System: macOS Catalina
 - Python Version: Python 3.7.6
 - Method of MultiQC installation: miniconda

**Comment by ewels** [comment]
Hi @eanoz,

Thanks for reporting this. It's a really annoying problem that comes up with Java tools sometimes, due to the localisation settings that you have on your system. Unlike most other tools, Java programs respect this and use `,` instead of `.` as the decimal separator. This is a nightmare for MultiQC as it's impossible to know whether `123,456` is the same as `123.456` or `123456` because `,` is used as a thousands separator in British locale, for example.

So a quick fix for you is to change the locale settings wherever you run QoRTs. Slower and more permanent fix is for me to add some horrible hacks to the MultiQC code to try to guess this. I'm pretty sure I've done this elsewhere, so I just need to find that code.

Phil

**Comment by ewels** [comment]
Hi @eanoz,

Thanks again for this. It's now fixed in MultiQC v1.9dev. Let me know if you run into any problems with it.

Phil
